workbench-core/accounts/src/models/accountSchema.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { z } from 'zod';
+import { HOSTING_ACCOUNT_STATUS } from '../constants/hostingAccountStatus';
+
+// eslint-disable-next-line @rushstack/typedef-var
+const ErrorSchema = z.object({
+  type: z.string(),
+  value: z.string()
+});
+
+// eslint-disable-next-line @rushstack/typedef-var
+export const AccountSchema = z.object({
```

Comment:
Why is this a `const` and not an `interface`?

Reply:
Why do we have this file and `account.ts`? Aren't tehy defining the same thing?

Reply:
`AccountSchema` isn't an `interface`, it is a Zod parser that parses and validates our data into the `Account` interface format.